lldb/test/API/lang/swift/clangimporter/dynamic_type_resolution_import_conflict/TestSwiftDynamicTypeResolutionImportConflict.py

```diff
@@ -69,4 +69,4 @@ def test(self):
         self.expect("fr v -d run-target -- input",
                     substrs=['(Dylib.LibraryProtocol) input'])
         self.expect("expr -d run-target -- input",
-                    substrs=['(Dylib.LibraryProtocol)'])
+                    substrs=['(a.FromMainModule)'])
```

Comment:
how come this changed?

Reply:
Reflection has better information when compared to remote AST in this case, so now we get the dynamic class type instead of just the protocol (same with `TestSwiftPrivateImport`)